specifyweb/workbench/upload/tests/testuploading.py

```diff
@@ -173,15 +173,15 @@ def test_enforced_county(self) -> None:
             results[0].record_result,
             FailedBusinessRule(
                 message='missingRequiredTreeParent',
-                payload={'names':['County']},
+                payload={'names':"['County']"},
                 info=ReportInfo(tableName='Geography', columns=['City'], treeInfo=None)
             )
         )
         self.assertEqual(
             results[1].record_result,
             FailedBusinessRule(
                 message='missingRequiredTreeParent',
-                payload={'names':['County']},
+                payload={'names':"['County']"},
```

Comment:
This doesn't look right. Front-end expects this to be a list, not a json string

Reply:
the payload value is an array now.  This change fixed that: [treerecord.py](https://github.com/specify/specify7/blob/eb377cae961fe8eb23b292173fcf401d782e5347/specifyweb/workbench/upload/treerecord.py#L297)